docs/getting-started.mdx

```diff
@@ -0,0 +1,237 @@
+
+<figure>
+  <HowItWorksDiagram />
+  <figcaption>
+    During the app development you install a capability and OneSDK. In the
+    runtime the instructions for calling the API are downloaded from the
+    Superface registry and the API is called directly from your app.
+  </figcaption>
+</figure>
+
+## Superface in 3 minutes
+
+The following steps will guide you how to quickly get started with Superface and OneSDK.
+
+:::info Prerequisites
+
+This guide assumes you have [Node.js](https://nodejs.dev/) version 14 or newer installed.
+
+:::
+
+<div class="count-steps-h3">
+
+### Pick a capability
+
+Decide what capability you would like to use in your App. A capability is described by its profile with unique ID.
+
+In this guide we will use the capability **List user repositories of a VCS** like GitHub or GitLab. Its ID is `vcs/user-repos`.
+
+:::note Capability details
+
+To see a detailed description of the capability, check the [`vcs/user-repos` profile page](https://superface.ai/vcs/user-repos).
+
+You can discover more capabilities at [superface.ai](https://superface.ai/catalog).
+
+:::
+
+### Pick a provider
+
+The next step is to **pick a provider you would like to use**. In this guide we will use the `github`.
+
+:::note Additional providers
+
+Find the list of supported capability providers at [`vcs/user-repos` profile page](https://superface.ai/vcs/user-repos).
+
+:::
+
+### Install OneSDK
+
+In your App directory, install the OneSDK library (`@superfaceai/one-sdk`):
+
+```shell
+npm install @superfaceai/one-sdk
+```
+
+### Install the capability
+
+Next, add the `vcs/user-repos` capability to your App using the Superface CLI (`@superfaceai/cli`):
+
+```shell
+npx @superfaceai/cli install vcs/user-repos
+```
+
+### Configure the provider
+
+The next step is to configure the provider. In our case we configure `github` for capability profile `vcs/user-repos`:
+
+```shell
+npx @superfaceai/cli configure github --profile vcs/user-repos
+```
+
+:::note Provider authentication
+In many cases you also need to set up provider credentials. Since we
+are using the unauthenticated GitHub API we can skip it for now.
+
+<!-- TODO:
+To learn how to authenticate providers check the provider authentication.
+-->
+
+:::
+
+### Code
+
+Finally, use OneSDK in your App. Create the `index.js` file with the following JavaScript code:
+
+```js title="index.js"
+const { SuperfaceClient } = require('@superfaceai/one-sdk');
+const sdk = new SuperfaceClient();
+
+async function main() {
+  // Load the capability profile
+  const profile = await sdk.getProfile('vcs/user-repos');
+
+  // Invoke the capability use case
+  const result = await profile.getUseCase('UserRepos').perform({
```

Comment:
This is again just a feeling, but chaining the methods on newlines seems more readable.

Reply:
Good point, Prettier insists on this form so I am opting it out for this code block.